src/Sarif/Autogenerated/LogicalLocationEqualityComparer.cs

```diff
@@ -43,7 +43,7 @@ public bool Equals(LogicalLocation left, LogicalLocation right)
                 return false;
             }
 
-            if (left.ParentKey != right.ParentKey)
+            if (left.ParentIndex != right.ParentIndex)
             {
                 return false;
             }
```

Comment:
>} [](start = 12, length = 1)

Again, not necessarily correct when comparing logical location objects from different runs. We shall see how this plays out.

Reply:
It is critical that result matching travels the complete parent chain. It is also critical that ParentIndex itself is not part of our value type comparison semantics (but it is today!)
[https://github.com/Microsoft/sarif-sdk/issues/1175](https://github.com/Microsoft/sarif-sdk/issues/1175)
[https://github.com/Microsoft/sarif-sdk/issues/1176](https://github.com/Microsoft/sarif-sdk/issues/1176)

---
In reply to: [243719340](https://github.com/Microsoft/sarif-sdk/pull/1170#discussion_r243719340) [](ancestors = 243719340)